### PR TITLE
Update synth scripts to remove jsondoc from rakefiles

### PR DESCRIPTION
**Pull request by dazuma**
These are the synth scripts that generate Rakefiles. Include a transform that removes the jsondoc related tasks.

### CODE REVIEW

**Comment by quartzmo** [comment]
Can you also remove this line from `Gemfile`?

```rb
gem "gcloud-jsondoc",
    git: "https://github.com/GoogleCloudPlatform/google-cloud-ruby.git",
    branch: "gcloud-jsondoc"
```

File: google-cloud-asset/synth.py [hunk 1]
```diff
@@ -67,3 +67,13 @@ def escape_braces(match):
     'lib/google/cloud/asset/v1beta1/asset_service_client.rb',
     '\n\n(\\s+)class OperationsClient < Google::Longrunning::OperationsClient',
     '\n\n\\1# @private\n\\1class OperationsClient < Google::Longrunning::OperationsClient')
+
+# https://github.com/googleapis/gapic-generator/issues/2278
+s.replace(
```
Comment (quartzmo) [inline google-cloud-asset/synth.py]: There is also a description line above:

```rb
desc "Generates JSON output from google-cloud-tasks .yardoc"
```

Comment (quartzmo) [reply google-cloud-asset/synth.py]: nm, i just saw it is included in the pattern, sorry!

**Comment by dazuma** [comment]
@quartzmo I'm pretty sure none of the synth scripts are using generated Gemfiles.

**Pull request merged**